ACKategories/UIApplicationExtensions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UIApplicationExtensions.swift
+//  ACKategories
+//
+//  Created by Igor Rosocha on 12/16/19.
+//  Copyright © 2019 Ackee, s.r.o. All rights reserved.
+//
+
+import UIKit
+
+public extension UIApplication {
+    /// Completely clear launch screen cache
```

Comment:
Please add a short story what is this method for and when and why should I use it and also the link to article 🙏

Reply:
No problem 🙂